compiler/rustc_middle/src/mir/mod.rs

```diff
@@ -1592,14 +1592,14 @@ impl<'tcx> Place<'tcx> {
         self.projection.iter().any(|elem| elem.is_indirect())
     }
 
-    /// If MirPhase >= Derefered and if projection contains Deref,
-    /// It's guaranteed to be in the first place
-    pub fn has_deref(&self) -> bool {
+    /// If MirPass >= Derefered and if projection contains Deref,
```

Comment:
`MirPass` doesn't implement `PartialOrd` so this doesn't make a lot of sense.

`MirPhase` *does* implement `PartialOrd`, on the other hand.